### ISSUE TITLE
save ss_summary as artifact

### ISSUE THREAD

**Issue by Rick-Methot-NOAA** [nmfs-stock-synthesis/ss-test-models#14]
A recent pull request failed, but a comparison of the failed quantities showed no differences when run on my local Windows machine.  Is it possible to access the ss_summary.sso files from the github flow runs?  Or perhaps to save that file from the "simple" run as an artifact?

**Comment by kellijohnson-NOAA** [comment]
@Rick-Methot-NOAA how soon do you need this? I know @k-doering-NOAA in on AL the rest of this week. It looks like I can change the code to at least copy the simple ss_summary.sso file over for you if you need it immediately. Though @k-doering-NOAA is really the expert here wrt these workflow files.

**Comment by Rick-Methot-NOAA** [comment]
Thanks Kelli.  If this is easy for you, I could use the base and test
ss_summary files from simple for the most recent pull request.  But no
rush.  It can wait til next week.

**Comment by kellijohnson-NOAA** [comment]
@Rick-Methot-NOAA I made the change that I "think" is needed. I am currently running it on the main branch to see if it works and if it successfully saved the file, then I will start it on your pull request.

**Comment by kellijohnson-NOAA** [comment]
@Rick-Methot-NOAA my second attempt was successful (nmfs-stock-synthesis/workflows@86606af) and the file for the main branch is available. I am running the workflow for the pull request right now. I think we should leave this issue open in case @k-doering-NOAA wants to make any changes.

**Comment by Rick-Methot-NOAA** [comment]
Thanks Kelli.
A character - by - character comparison between this ss_summary and the ss_summary produced on my local machine showed identical parameter estimates with identical parameter variance.   This difference is in variance of the derived quantities and now  I see that variance of derived quantities is flawed on my local machine also.
Must have messed up the order of ops somewhere in equil_calcs.   :(

**Comment by k-doering-NOAA** [comment]
Thanks @kellijohnson-NOAA for modifying the workflow!

@Rick-Methot-NOAA, for the long term, we can certainly save the summary files, if they turn out to be helpful - I was just trying to keep the artifact size down, but if we need more output, I think that should work fine!

**Comment by Rick-Methot-NOAA** [comment]
Turns out that I did not need the file, but needed it this once in order to
come to that realization.
I routinely do a character-by-character compare for at least one ss_summary
and sometimes for the report.sso to be sure nothing gets missed by the
numerical compares we do.
In that regard, I think it better to do comparisons for the last year of
the forecast, not the first year, as I see that some differences don't
creep in until forecast recruitments enter the population.
